packages/react-native-test-renderer/src/renderer/index.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow
+ */
+
+'use strict';
+
+import type {Element, ElementType} from 'react';
+
+import * as FabricUIManager from './FabricUIManagerMock';
+
+import {act} from 'react-test-renderer';
+import ReactFabric from 'react-native/Libraries/Renderer/shims/ReactFabric';
+
+type ReactNode = {
```

Comment:
Would it be possible to provide `toJson()` on React Node, so that users could do focussed snapshots on only a part of the element tree?